pkg/aws/eni/node_manager_test.go

```diff
@@ -213,6 +215,8 @@ func updateCiliumNode(cn *v2.CiliumNode, available, used int) *v2.CiliumNode {
 
 func reachedAddressesNeeded(mngr *ipam.NodeManager, nodeName string, needed int) (success bool) {
 	if node := mngr.Get(nodeName); node != nil {
+		//fmt.Println(node)
+		fmt.Println(node.GetNeededAddresses())
```

Comment:
I think these debug lines can be removed.

Reply:
indeed, done 👌